### PR TITLE
Add component of footer for index page

### LINKED ISSUE #2: Create component of footer for the index page
Attaching screenshot for reference.

Screenshot (155)

### PR DESCRIPTION
**Pull request by himanshuvarandani**
- [x] Closes #2

---

## PR description

Add responsive footer component for index page. Also extend a class for blue color hex code #003366 as "bg-blue-850".

Screenshots of footer for some screen size are shown below:

For screen size > 1024,
image

For screen size > 768,
image

image

For screen size > 640,
image

For screen size < 640,
image

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Link from "next/link"
+
+export default function Footer() {
+  return (
+    <div className="grid sm:grid-cols-8 md:grid-cols-10 lg:grid-cols-12 text-white mt-5">
+      <div className="sm:col-span-2 flex items-center justify-center p-2">
+        <img src="/logo.png" className="max-h-24" />
```
Comment (pnijhara) [inline components/Footer.js:7]: Do not use \. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element.  @next/next/no-img-element

**Comment by pnijhara** [comment]
Also, the footer in mobile view does not look good. You can add social media links on the right side in mobile view. Also, you can remove the image on the mobile as it is taking a lot of space. You can also centre align the legal part.

File: tailwind.config.js [hunk 2]
```diff
@@ -2,10 +2,16 @@ module.exports = {
   purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        blue: {
+          850: "#003366",
```
Comment (pnijhara) [inline tailwind.config.js:8]: @sanjana-302, check this out.

**Comment by pnijhara** [comment]
@himanshuvarandani, you can write a note about it in the readme since you have changed the tailwind config file for the blue colour.

**Comment by sanjana-302** [comment]
Looks good to me. Can we add for red too?

**Comment by pnijhara** [comment]
> Looks good to me. Can we add for red too?

Maybe later in the header PR.

**Comment by roshnet** [comment]
Changes look good, merging!

**Pull request merged**